lib/patch-series.ts

```diff
@@ -328,23 +328,19 @@ export class PatchSeries {
 
     protected static encodeSender(sender: string): string {
         const encoded = encodeWords(sender);
-        const dot = encoded.indexOf(".");
-        if (dot < 0) {
+
+        /* Don't quote if already quoted */
+        if (encoded.startsWith("\"") && encoded.indexOf("\" <") > 0) {
```

Comment:
I would have expected this to match whitespace loosely (just like the check later in the patch for a _special_ character in the name):

```
    ... && encoded.match(/"\s*</) {
```

Reply:
Makes sense! Thanks.